test-vectors/determinism-check

```diff
@@ -1 +1 @@
-- 81b8e58b84a114ef13064115cecd3af249e588cd2ec7100f34528838df31c5c7
```

Comment:
I'm surprised that this changed at all -- does this make sense to you? Is the addition of the new method somehow noticeable by the vectors?

Reply:
Did this only change after the sector key addition?  If you don't remember go back and check the commit before you made that change.  I would expect sector key addition to change everything since on chain sector info format is different.  I wouldn't expect introducing a new method and no new tests without state changes to change this.

Reply:
D'oh, forgot about the sector key. Confirmed this only changed after that was added, so is as expected.